app/views/contexts/hub_context.py

```diff
@@ -51,24 +59,32 @@ def get_context(self, survey_complete, enabled_section_ids) -> Mapping:
                 "button"
             ) or lazy_gettext("Submit survey")
 
+            individual_response_enabled = False
+
         else:
             title = lazy_gettext("Choose another section to complete")
+
             guidance = custom_text.get("guidance") or lazy_gettext(
                 "You must complete all sections in order to submit this survey"
             )
+
             submit_button = lazy_gettext("Continue")
 
+            individual_response_enabled = self._individual_response_enabled(
+                self._schema
+            )
+
         return {
             "title": title,
             "guidance": guidance,
+            "individual_response_enabled": individual_response_enabled,
```

Comment:
Is this used?. I can't see any usages of this in `hub.html`.

Reply:
It will be when the next card goes in 😄